NetShared/StormTypes.h

```diff
@@ -1,16 +1,23 @@
 #pragma once
 #include "../shared_common.h"
+#include "../SNP/include/base64.hpp"
+
+#ifndef _WIN32
+u32 GetTickCount() {
+    return std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::steady_clock::now().time_since_epoch()).count();
```

Comment:
We want integer epoch we would probably use `time()` from `<cstdlib>`, or the alternative from C++, this chrono clock is high performance clocks, you would want `system_clock` probably. Also there is no point in converting to ms if you just want to replicate GetTickCount, since time_since_epoch return integer value.

As an aside if you find it interesting, there were tricks to use on the previous code:

If you do `using namespace std::literals`

You should be able to more concisely express this as
`return std::chrono::steady_clock::now().time_since_epoch() / 1ms`

I typically use `high_precision_clock`, altho those are usually the same. It's usually better to use time in seconds than milliseconds. Unless we really badly need to. It's easier to do math on.